config/initializers/rack_timeout.rb

```diff
@@ -0,0 +1,2 @@
+Rack::Timeout.service_timeout = (ENV["RACK_SERVICE_TIMEOUT_SECONDS"] || 10).to_s
+Rack::Timeout.wait_timeout    = (ENV["RACK_WAIT_TIMEOUT_SECONDS"]    || 10).to_s
```

Comment:
I'm not sure what value to set for the wait timeout.  According to [Heroku's docs](https://github.com/heroku/rack-timeout):

> When a request has a wait time greater than wait_timeout, it'll be dropped without ever being sent down to the application, and a Rack::Timeout::RequestExpiryError is raised. Such requests are logged as expired.

So, I think we need to set this to be whatever wait time is configured in our infrastructure.

What I don't know (and we can discuss in Slack), is how our infrastructure is timing out requests.  Does it just kill any request after N seconds regardless of what I/O is happening, or is it more like Heroku's where it kills the request only if there's nothing coming.